client/app/telecom/sms/options/response/telecom-sms-options-response.html

```diff
@@ -175,99 +175,41 @@ <h1 data-translate="sms_options_response"></h1>
-                                            role="menu"
-                                            aria-labelledby="btn-sms-options-response-action-{{ index }}"
-                                            data-uib-dropdown-menu>
-                                            <li role="menuitem">
-                                                <i class="ovh-font ovh-font-editer" aria-hidden="true"></i>
-                                                <button
-                                                    type="button"
-                                                    class="btn btn-link"
-                                                    data-ng-click="TelecomSmsOptionsResponseCtrl.editTrackingOptions($index, option)"
-                                                    data-translate="edit">
-                                                </button>
-                                            </li>
-                                            <li role="menuitem">
-                                                <i class="ovh-font ovh-font-bin" aria-hidden="true"></i>
-                                                <button
-                                                    type="button"
-                                                    class="btn btn-link"
-                                                    data-ng-click="TelecomSmsOptionsResponseCtrl.removeTrackingOptions($index, option)"
-                                                    data-translate="delete">
-                                                </button>
-                                            </li>
-                                        </ul>
-                                    </div>
-                                </td>
-                            </tr>
-                        </tbody>
-                        <tbody data-ng-if="TelecomSmsOptionsResponseCtrl.loading.init">
-                            <tr>
-                                <td colspan="3">
-                                    <oui-spinner></oui-spinner>
-                                </td>
-                            </tr>
-                        </tbody>
-                        <tbody data-ng-if="!TelecomSmsOptionsResponseCtrl.loading.init && !TelecomSmsOptionsResponseCtrl.smsResponse.trackingOptions.length">
-                            <tr>
-                                <td class="text-center"
-                                    colspan="3"
-                                    data-translate="sms_options_response_tracking_empty">
-                                </td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </div>
+                <oui-datagrid class="reduce-padding" rows="TelecomSmsOptionsResponseCtrl.smsResponse.trackingOptions">
+                    <oui-column title="'sms_options_response_tracking_media_label' | translate" property="media" sortable>
```

Comment:
According to CX guidelines, all grid should have a default sorted column. In most cases we'll have to add `sortable="asc"` on the first column.